sse2neon.h

```diff
@@ -8358,6 +8359,645 @@ FORCE_INLINE int _mm_testz_si128(__m128i a, __m128i b)
 
 /* SSE4.2 */
 
+enum {
```

Comment:
The `_MM_CMPINT_ENUM` type used to specify comparison operations in AVX-512 intrinsics. Since we are concentrating on SSE 4.2 intrinsics, we can safely remove this enumeration.

Reply:
Check the headers shipped in LLVM and GCC, and you will figure out that the definition appears in `<avx512fintrin.h>`.